schema/Retail/Cancel/onCancel.ts

```diff
@@ -242,7 +242,7 @@ export const onCancelSchema = {
                           code: {
                             type: 'string',
                             minLength: 1,
-                            const: 'Cancelled',
+                            enum: ['Cancelled', 'RTO-Initiated', 'RTO-Delivered', 'RTO-Disposed'],
```

Comment:
It will always be RTO-Initiated in on_cancel. RTO will be disposed or delivered later using /on_status call.

Reply:
Understood